### PR TITLE
Feature/support https

### PR DESCRIPTION
**Pull request by kikkomep**

### CODE REVIEW

File: portal/values.yaml [hunk 1]
```diff
@@ -27,7 +28,7 @@ frontend:
   service:
     name: http
     type: NodePort
-    externalPort: 30766
+    externalPort: 30767
```
Comment (pcm32) [inline portal/values.yaml:31]: Any reason for the port change? might mean asking a different URL to be whitelisted at EBI Portal...

**Comment by pcm32** [comment]
LGTM besides the port thingy... I would avoid changing the port number unless really needed.

File: portal/values.yaml (hunk 1 above)
Comment (kikkomep) [reply portal/values.yaml]: We are already using the port 30767 on portaldev.

Comment (kikkomep) [reply portal/values.yaml:31]: It is more practical. We are actually using the 30767 port on portaldev and it is also the port allowed for development purposes on minikube. The main installation of the portal can be configured to use the 30766 port using the config file.

**Comment by pcm32** [comment]
Ahhh, I remember now, one was for portal and the other for portaldev... sorry, didn't remember that.

**Pull request merged**